vulnerabilities/templates/api_user_creation_form.html

```diff
@@ -11,15 +11,33 @@
           </div>
         </article>
       {% endfor %}
-    {% block title %}
-    VulnerableCode API key request
-    {% endblock %}
+      <div id="form-errors" class="message is-danger {% if not form.errors %}is-hidden{% endif %}">
+        {% for field_name, errors in form.errors.items %}
+        <div class="message-body">
+          {{ errors }}
+        </div>
+        {% endfor %}
+      </div>
+    <h2 class="subtitle mb-0 pt-2 mb-2">
+      <b>VulnerableCode API key request</b>
+    </h2>
+
+    <section class="ml-2 mt-3 pr-4 pt-4 pl-4 pb-4 has-background-light has-border-radius">
```

Comment:
An additional paragraph, why one needs an API key and what that API will provide, would be nice.

Reply:
Done!